src/main/resources/org/csanchez/jenkins/plugins/kubernetes/KubernetesCloud/help-pspSecurityContext.html

```diff
@@ -0,0 +1,26 @@
+<div>
+    <p>
+        When using <a href="https://kubernetes.io/docs/concepts/security/pod-security-admission/">Pod Security Admission</a> in the Agents namespace
+        with the <code>restricted</code> <a href="https://kubernetes.io/docs/concepts/security/pod-security-standards/">Pod Security Standard</a>, the
+        <code>jnlp</code> container cannot be scheduled without overriding its container definition with a <code>securityContext</code>.
+    </p>
+    <p>
+        This option allows to automatically inject in the <code>jnlp</code> container a <code>securityContext</code> that is suitable for the use
+        of the <code>restricted</code> <a href="https://kubernetes.io/docs/concepts/security/pod-security-standards/">Pod Security Standard</a>.
+    </p>
+    <p>
+        This is the <code>securityContext</code> that will be used for the <code>jnlp</code> container with that option:
+    </p>
+    <pre>
+        <code>
+            securityContext:
```

Comment:
Check rendering; I suspect this would be indented much too much in the GUI.

Reply:
Rendering is illustrated in the PR description, it was indeed probably a bit too much indented. I updated to:

![image](https://github.com/jenkinsci/kubernetes-plugin/assets/311677/adab1c0c-d0fe-49e4-8ffc-3115ac42551b)